runner/runners/polling.go

```diff
@@ -48,6 +48,10 @@ func (r *PollingStatusQuerier) Query(q runner.Query, wait runner.Wait) ([]runner
 			return st, service, err
 		}
 		time.Sleep(r.period)
+
+		// Exponentially increase time between polls
+		r.period *= 2
```

Comment:
This backoff is changing the state of the PollingStatusQuerier. Is that by design? I think you only want to increase the period for a specific method invocation, which means introducing a variable on the local call stack.

Reply:
+1

Reply:
also think we should have a max value ?3 min?

Reply:
I set the max at 10s. Jeanette and I talked about the limit yesterday and agreed on 10s as suggested by Drew in another comment.